tools/alpine/Makefile

```diff
@@ -2,24 +2,36 @@
 
 ORG?=linuxkit
 IMAGE=alpine
+
+DOCKER_CONTENT_VAR=1
+PKG_DEPS=packages
+ifeq ($(shell uname -m), x86_64)
 BASE=alpine:3.6
+PKG_DEPS += packages.x86_64
+endif
+
+ifeq ($(shell uname -m), aarch64)
+BASE=arm64v8/alpine:3.6
+PKG_DEPS += packages.aarch64
+DOCKER_CONTENT_VAR=0
+endif
```

Comment:
I lost track of this part, but is there a reason, this is not following the pattern we have elsewhere (`pkg/package.mk`) and simply just sets `NOTRUST` when on arm?